src/adapters/api-blueprint-adapter.coffee

```diff
@@ -67,6 +67,48 @@ ensureObjectOfObjects = (data, key = 'name') ->
     data
 
 
+# # Merges multiple headers with the same key into one
+# This is needed in A1 -> legacy header merge / transformation,
+# since A1 can contain multiple values for given
+# header key, legacy cannot
+#
+#   [
+#     { name: 'Set-Cookie', value: 'abcde' }
+#     { name: 'Set-Cookie', value: 'Alan Turing' }
+#   ]
+#
+# is turned into
+#
+#   [
+#     { name: 'Set-Cookie', value: 'abcde; Alan Turing' }
```

Comment:
Note, `;` is already used as a token separator in both `Set-Cookie` and `Link` headers.

Example, the following `Link` headers are equivalent:

```
Link: <https://api.github.com/user/repos?page=3&per_page=100>; rel="next",
      <https://api.github.com/user/repos?page=50&per_page=100>; rel="last"
```

```
Link: <https://api.github.com/user/repos?page=3&per_page=100>; rel="next"
Link: <https://api.github.com/user/repos?page=50&per_page=100>; rel="last"
```

Whereas joining them by `;` will change the definition.